modular_skyrat/modules/importedvendors/readme.md

```diff
@@ -0,0 +1,26 @@
+## Title: ImportedVendors
```

Comment:
**Question for Maintainers to have input on before the author acts on it**
---
Could this be a sub-module of modular_skyrat\modules\modular_vending? I know we're trying to look into how many modules we have, and sorting certain things into sub-modules could help

I.e. 
modular_skyrat\modules\modular_vending\code (for the unique ones or overrides)
modular_skyrat\modules\modular_vending\importedvendors (for this)

Reply:
It could be a submodule, yeah. That wouldn't be a bad idea at all, actually. I like it. So long as the name of the folder is changed to `imported_vendors`, I'm all for it.